org.adempiere.ui.zk/js/ckeditor/config.js

```diff
@@ -1,21 +1,11 @@
 CKEDITOR.editorConfig = function(config) {
     config.resize_enabled = false;
     config.toolbarCanCollapse = true;
-    config.toolbar = 'MyToolbar';
     config.coreStyles_bold = { element: 'b', overrides: 'strong' };
     config.coreStyles_italic = { element: 'i', overrides: 'em' };
     config.coreStyles_strike = { element: 'strike', overrides: 's' };
-    config.toolbar_MyToolbar =
-[
-	{ name: 'styles', items : [ 'Styles','Format','Font','FontSize' ] },
-	{ name: 'colors', items : [ 'TextColor','BGColor' ] },
-	{ name: 'tools', items : [ 'Maximize', '-','About' ] },
-	{ name: 'clipboard', items : [ 'Undo','Redo' ] },
-	{ name: 'editing', items : [ 'Find','Replace','-','SelectAll' ] },
-	'/',
-	{ name: 'basicstyles', items : [ 'Bold','Italic','Underline','Strike','Subscript','Superscript','-','RemoveFormat' ] },
-	{ name: 'paragraph', items : [ 'NumberedList','BulletedList','-','Outdent','Indent','-','Blockquote','-','JustifyLeft','JustifyCenter','JustifyRight','JustifyBlock' ] },
-	{ name: 'links', items : [ 'Link','Unlink' ] },
-	{ name: 'insert', items : [ 'Image','Table','HorizontalRule','SpecialChar' ] }
-];
+    config.toolbar = [
+		{ name: 'styles', items : [ 'Styles','Format','Font','FontSize' ] },
+		{ name: 'colors', items : [ 'TextColor','BGColor' ] },
+	];
```

Comment:
this change is not right. the idea is config-min.js is for simple/minimum toolbar and config.js for the full toolbar.

Reply:
do we have to show all toolbar buttton? The current toolbar button is too much and not neccessary for basic user when writing email message.

Reply:
I means if we were to show a basic toolbar, we should just use the config-min.js instead of removing config-min.js and modify config.js. That would effect other parts of the application.

Reply:
I'm sorry. I mischeck when searching which use ckeditor/config.js.